e2e-tests/commands/shared-commands/assertions/assert-yes-radio-option-is-checked.js

```diff
@@ -0,0 +1,11 @@
+import { yesRadioInput } from '../../../pages/shared';
+
+/**
+ * assertYesRadioOptionIsChecked
+ * Assert that a "yes" radio option is checked.
+ */
+const assertYesRadioOptionIsChecked = () => {
```

Comment:
how will this work for more than one yes/no radio on the page

Reply:
what if we pass the number, so by default it's input, but if you pass a number, then it does `yesRadioInput.eq(number)`? I think all yes no radios should come to this command rather than some going straight to `assertRadioOptionIsChecked` and some to here?
Opinion?

Reply:
```js
cy.assertRadioOptionIsChecked(yesRadioInput().first());
cy.assertRadioOptionIsChecked(yesRadioInput().eq(1));
```

Reply:
@Zainzzkk Great idea, have added `index` and `eq()` logic to `cy.assertRadioOptionIsChecked()`.